mobile_files/package.json

```diff
@@ -20,6 +20,7 @@
     "create-react-class": "15.6.2",
     "dns.js": "1.0.1",
     "emojilib": "2.2.9",
+    "eth-phishing-detect": "~1.1.13",
```

Comment:
Why the version with ~ ?

Reply:
They update eth-phishing-detect database quite frequently, and it's better to have this updates in time. I use `~` here to have only patch versions fetched and used in Status.

Reply:
We want to have reproducible builds so we must fix the version. It's fine if we have to update this frequently.